src/swarm/neo/authentication/HmacAuthCode.d

```diff
@@ -109,10 +109,7 @@ struct HmacAuthCode
     import cstdio = core.stdc.stdio: fprintf, stderr;
     import core.stdc.stdlib: exit, EXIT_FAILURE;
 
-    version (D_Version2)
-        mixin("shared static this ( ) { staticCtor(); }");
-    else
-        static this ( ) { staticCtor(); }
+    shared static this ( ) { staticCtor(); }
```

Comment:
This could be refactored to include the code that's inside `staticCtor`. I don't think having that separate function is needed any more.

Reply:
I added a release note for this.

Reply:
Done!